cmd/cache/add.go

```diff
@@ -51,9 +62,10 @@ var addCmd = &cobra.Command{
 
 func init() {
 	CacheCmd.AddCommand(addCmd)
-	addCmd.Flags().StringVarP(&region, "region", "r", "", "The region to use for the AWS S3 cache")
+	addCmd.Flags().StringVarP(&region, "region", "r", "", "The region to use for the AWS S3 or GCS cache")
 	addCmd.Flags().StringVarP(&bucketname, "bucket", "b", "", "The name of the AWS S3 bucket to use for the cache")
 	addCmd.MarkFlagsRequiredTogether("region", "bucket")
+	addCmd.Flags().StringVarP(&projectId, "projectid", "p", "", "The GCP project ID")
 	addCmd.Flags().StringVarP(&storageAccount, "storageacc", "s", "", "The Azure storage account name of the container")
 	addCmd.Flags().StringVarP(&containerName, "container", "c", "", "The Azure container name to use for the cache")
 	addCmd.MarkFlagsRequiredTogether("storageacc", "container")
```

Comment:
@matthisholleville , minor but you can add the projectId to exclude from being set with Azure cache arguments and thus fail fast the configuration of  a cache